db/migrations/0004_alter_order_created_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-11-18 11:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_order_created_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(blank=True, null=True),

```

Comment:
The `created_at` field in the `Order` model should be set with `auto_now_add=True` to automatically set the timestamp during creation. Allowing `null` and `blank` values contradicts the task requirements and should be avoided. Please ensure `auto_now_add=True` is included and remove `blank=True, null=True`.